test/support/builders/atom_watcher_event.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+
+const metadata = require('../../../core/metadata')
+const events = require('../../../core/local/steps/event')
+
+/*::
+import type { Stats } from 'fs'
+import type { AtomWatcherEvent, Action, Kind, Batch } from '../../../core/local/steps/event'
+*/
+
+function randomPick /*:: <T> */ (elements /*: Array<T> */) /*: T */{
+  const l = elements.length
+  const i = Math.floor(Math.random() * l)
+  return elements[i]
```

Comment:
```js
_.first(_.shuffle(elements))
```

Reply:
I'm not a big fan of shuffling the array just to pick one element.